zookeeper-server/src/test/java/org/apache/zookeeper/server/DataTreeTest.java

```diff
@@ -222,7 +222,7 @@ public void testPathTrieClearOnDeserialize() throws Exception {
         PathTrie pTrie = (PathTrie)pfield.get(dserTree);
 
         //Check that the node path is removed from pTrie
-        Assert.assertEquals("/bug is still in pTrie", "", pTrie.findMaxPrefix("/bug"));
+        Assert.assertEquals("/bug is still in pTrie", "/", pTrie.findMaxPrefix("/bug"));
```

Comment:
The root node is always a prefix.  This change puts this method in line with the rest of the code base which all makes this assumption.